### PR TITLE
LangChain LLM support

### PR DESCRIPTION
**Pull request by itaiMona**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,11 +220,24 @@ Since for streaming responses OpenAI doesn't supply the full usage tokens summar
 
 NOTE: Stream is currently only supported with SDK usage, and not with using REST directly.
 
-### Langchain support
+## LangChain support
 
-We currently only support monitoring OpenAI endpoints when using Langchain via monkey-patching. See completion_langchain_monkeypatch.py for an example.
+You can use the exported `monitor_langchain_llm` to wrap a LangChain OpenAI LLM (chat or normal) with Mona's monitoring capabilities.:
```
Comment (TalzMona) [inline README.md]: You have both "." and ":"

File: README.md [hunk 2]
```diff
@@ -219,11 +220,24 @@ Since for streaming responses OpenAI doesn't supply the full usage tokens summar
...
 
-A more robust Langchain integration is in the works.
+```py
+from mona_openai import monitor_langchain_llm
+
+from langchain.llms import OpenAI
+
+# Loading Mona credetntials etc...
```
Comment (TalzMona) [inline README.md]: Actually, I think we should mention explicitly what we are loading here

File: examples/completion/completion_langchain.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+MONA_API_KEY = environ.get("MONA_API_KEY")
+MONA_SECRET = environ.get("MONA_SECRET")
+MONA_CREDS = {
+    "key": MONA_API_KEY,
+    "secret": MONA_SECRET,
+}
+
+# This is the name of the monitoring class on Mona
```
Comment (TalzMona) [inline examples/completion/completion_langchain.py]: "." in end of sentence

**Review by TalzMona — APPROVED** [review]
left really small comments. LGTM

File: README.md (hunk 2 above)
Comment (itaiMona) [reply README.md]: Changed this completely. I was a mistake here.

**Pull request merged**